package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "body-parser": "^1.17.2",
     "bootstrap": "^3.3.7",
+    "bower": "^1.8.2",
```

Comment:
this shouldn't be needed anymore, https://github.com/ember-cli/ember-try/issues/161 got fixed tonight.

Reply:
👍